api/db/changelog/db.changelog-master.xml

```diff
@@ -158,7 +158,8 @@
   <include file="changelog/db.changelog-146-access-tier-table-and-cdr-version.xml"/>
   <include file="changelog/db.changelog-147-remove-cluster-create-retries.xml"/>
   <include file="changelog/db.changelog-148-remove-cluster-config.xml"/>
-  <include file="changelog/db.changelog-149-demographic-survey-add-disability-prefer-no-answer.xml"/>
+  <include
```

Comment:
My IntelliJ adjusts this formatting when I save. If this does not match our standards, please let me know, and I can look into my linter.